docs/src/modeling/forms/links/README.md

```diff
@@ -0,0 +1,37 @@
+# Verlinkung von Webseiten oder Dokumenten an Nutzeraufgabe 
+
+DigiWF stellt die Möglichkeit bereit als Prozessmodellierer dem Sachbearbeiter zusätzliche Informationen bereitzustellen.
+
+Die Links können dazu über eine Prozessvariable erstellt werden.
+
+Die Links werden unter dem Taskname angezeigt.
+
+## Konfiguration 
+
+Das Setzen der Links wird hier an hand von einem Inlinescript erzeugt: 
+
+```js
+var ArrayList = Java.type('java.util.ArrayList');
+var TaskExternalReference = Java.type('de.muenchen.oss.digiwf.task.TaskExternalReference');
+externalLinks = new ArrayList();
+externalLinks.add(new TaskExternalReference('url', '[Google](https://google.de/)'));
+externalLinks.add(new TaskExternalReference('zammad', '[Ticket 11004832](LHM11004832)'));
+externalLinks.add(new TaskExternalReference('mucsdms', '[Vorgang 1](COO.2150.307.2.41134)'));
+execution.setVariable('app_task_external_links', externalLinks);
+
```

Comment:
Wir müssen nochmal darüber reden, welche Syntax wir hier vorgeben. Ich störe mich noch etwas an den Java-Typen

Im Code sprechen wir auch eher von JSON:
https://github.com/it-at-m/digiwf-core/blob/dev/digiwf-task/digiwf-task-api/src/main/java/de/muenchen/oss/digiwf/task/TaskExternalReference.java#L11

Reply:
Die Klärung können wir aber hier rausziehen, wenn wir drüber schreiben, dass dies eine experimental API ist und sich das Handling noch ändern kann

Reply:
Bei der Migration auf Camunda 8 müssen sowieso alle Prozesse angepasst werden. Bedeutet, dass hier da nicht Mehraufwand entsteht.

Reply:
Die Frage ist, wie viel man anpassen muss und da sehe ich einen Unterschied. Aber ich würde die Diskussion hier auslagern. Wir können gerne mit der Lösung starten, wenn wir es als Experimental-API bezeichnen